EWL-4912 radio button

designerneil opened the pull request:
## Ticket(s)

**Jira Ticket**
- [EWL-4912: SG2 | Update checkbox atom](https://issues.ama-assn.org/browse/EWL-4912)

## Description
Adds checkbox atom

## To Test
- [ ] `gulp serve`
- [ ] visit http://localhost:3000/?p=atoms-radio-button
- [ ] Observe a group of radio buttons with a label Year Medical Student
- [ ] Click on any of the radio buttons
- [ ] Observe the radio is purple with a purple border

## Visual Regressions

backstop test. No failures

## Relevant Screenshots/GIFs
<img width="149" alt="screen shot 2018-04-27 at 2 45 49 pm" src="https://user-images.githubusercontent.com/2271747/39382017-b5b0821e-4a29-11e8-8521-3957c228ca10.png">



## Remaining Tasks
N/A


## Additional Notes
**This PR was branched off of the checkbox PR which shares many of the same syles.
https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/200**

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

designerneil committed:
EWL-4912: Add radio button atom

labbydev reviewed (changes requested):
Similar to others I'm seeing some discrepancies between the Zeplin and PatternLab:
* Label should be all caps and light gray
* option labels should be a dark gray color
* Circles are getting cut off when selected

**Pattern Lab**
<img width="200" alt="screen shot 2018-04-26 at 11 01 20 am" src="https://user-images.githubusercontent.com/1653723/39318368-91fa8ff4-4943-11e8-9994-5cf3047d3689.png">

**Zeplin**
<img width="229" alt="screen shot 2018-04-26 at 11 18 42 am" src="https://user-images.githubusercontent.com/1653723/39318391-9df5bc48-4943-11e8-89c5-2f4ca84223c4.png">

designerneil committed:
Merge branch 'develop' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4912-radio-button

# Conflicts:
#	styleguide/source/assets/js/form-items.js

designerneil committed:
Fix PR comments and suggestions
Label should be all caps and light gray
option labels should be a dark gray color
Circles are getting cut off when selected

labbydev commented:
Updates look good just need to resolve the merge conflicts

designerneil committed:
Updates radio button selected state

designerneil committed:
Merge branch 'develop' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4912-radio-button

# Conflicts:
#	styleguide/source/assets/js/form-items.js

designerneil committed:
Merge branch 'develop' of github.com:AmericanMedicalAssociation/ama-style-guide-2 into feature/EWL-4912-radio-button

# Conflicts:
#	styleguide/source/assets/js/form-items.js

nstriedinger commented:
This doesn't have the purple ring "bullseye" around it like the Zeplin screenshot?

![screen shot 2018-04-27 at 3 32 59 pm](https://user-images.githubusercontent.com/231467/39381506-5f75f1ca-4a30-11e8-95db-d3f7b2f940c4.png)
